robottelo/cli/org.py

```diff
@@ -168,15 +168,3 @@ def remove_user(cls, options=None):
         """Removes an user from an org"""
         cls.command_sub = 'remove-user'
         return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def set_parameter(cls, options=None):
-        """Create or update parameter for an organization."""
-        cls.command_sub = 'set-parameter'
-        return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def delete_parameter(cls, options=None):
```

Comment:
@oshtaier disagree. hammer.json should contain all the commands. And there's no point in copying same command implementation for each cli command too, if exactly the same implementation is already there in Base class and will be inherited - similar to create/update/list/info which are implemented once in Base class and that's it (well, mostly, with very few exceptions).